packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -101,8 +106,9 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
           </span>
         ) : (
           <Bell
-            hasUnreadMessages={Boolean(inbox.unreadMessageCount)}
             className={props.className}
+            hasUnreadMessages={Boolean(inbox.unreadMessageCount)}
+            onMouseEnter={handleBellOnMouseEnter}
```

Comment:
i don't think we want this

Reply:
we only have one current list of messages, so until we have separate lists we can't prefetch